### ISSUE TITLE
Выделение мест растягиванием [4]

### ISSUE THREAD

**Issue by AndreyTsvetkov** [omega-code/place-selector#7]
Выделяем места не кликая по ним, а — в режиме выделения — растягивая прямоугольник выделения. Места, полностью попавшие внутрь него — выделяются. 

Повторное растягивание прямоугольника на других местах не сбрасывает предыдущее выделение. Повторное выделение уже выделенного места инвертирует его статус (то есть выделенное становится невыделенным и наоборот).

**Comment by AndreyTsvetkov** [comment]
при этом выделение кликом можно сохранить, если его сохранение не приведет к значительному усложнению кода, или убрать, если можно этим будет значительно упростить систему

**Comment by Traveler59** [comment]
Вопрос: стоит ли переименовать `Rect` в `Seat` и добавить логику сидения, вроде того что оно может быть выбрано или иметь номер, или создать дополнительный класс и сделать `Rect` его полем? Или пока оставить так?

**Comment by AndreyTsvetkov** [comment]
Да, хорошая идея разнести прикладное понятие "Посадочное место в зале" `Seat` и техническое "Прямоугольник на экране" `Rect` с его цветом/стилем заливки, всем тем, что не имеет значения кроме как для рисования. Так что вариант 

![text](http://yuml.me/diagram/normal/class/
  [Seat]++-1>[Rect])

мне нравится более всего.
